docs/notebooks/Custom_derivative_rules_for_Python_code.ipynb

```diff
@@ -2174,7 +2174,7 @@
       "source": [
         "### Working with `list` / `tuple` / `dict` containers (and other pytrees)\n",
         "\n",
-        "You should expect standard Python containers like lists, tuples, namedtuples, and dicts to just work, along with nested versions of those. In general, any [pytrees](https://github.com/google/jax/blob/master/docs/notebooks/JAX_pytrees.ipynb) are permissible, so long as their structures are consistent according to the type constraints. \n",
+        "You should expect standard Python containers like lists, tuples, namedtuples, and dicts to just work, along with nested versions of those. In general, any [pytrees](https://github.com/google/jax/blob/master/docs/pytrees.rst) are permissible, so long as their structures are consistent according to the type constraints. \n",
```

Comment:
We should link to the rendered doc instead: https://jax.readthedocs.io/en/latest/pytrees.html

Once we convert this content to myst-nb (see #5426) we can use sphinx cross-references to automatically keep these kinds of links up to date.

Reply:
It looks like https://github.com/google/jax/pull/5426 has been merged. Since, https://github.com/google/jax/blob/master/docs/notebooks/JAX_pytrees.ipynb returns a 404 in the notebook:

```
You should expect standard Python containers like lists, tuples, namedtuples, and dicts to just work, along with nested versions of those. In general, any [pytrees](https://github.com/google/jax/blob/master/docs/notebooks/JAX_pytrees.ipynb) are permissible, so long as their structures are consistent according to the type constraints. 
```

should this PR be LGTMed? Cheers

Reply:
The new version still points to the RST file rather than the renderered HTML. I think we should fix that before merging.

Reply:
Since this PR has stalled, I made the change in https://github.com/google/jax/pull/6808